src/client/Microsoft.Identity.Client/AuthScheme/PoP/PopAuthenticationOperation.cs

```diff
@@ -86,6 +88,14 @@ public void FormatResult(AuthenticationResult authenticationResult)
             authenticationResult.AccessToken = popToken;
         }
 
+        public Task FormatResultAsync(AuthenticationResult authenticationResult, CancellationToken cancellationToken = default)
```

Comment:
should this be an async method?

Reply:
The "async" modifier only applies if you have an active `await` inside the code. It isn't the case here.